tokio-console/src/state/tasks.rs

```diff
@@ -170,14 +194,17 @@ impl TasksState {
                 // remap the server's ID to a pretty, sequential task ID
                 let id = ids.id_for(span_id);
 
-                let short_desc = strings.string(match name.as_ref() {
-                    Some(name) => format!("{} ({})", id, name),
-                    None => format!("{}", id),
+                let short_desc = strings.string(match (task_id, name.as_ref()) {
+                    (Some(task_id), Some(name)) => format!("{} ({})", task_id, name),
```

Comment:
style nit, take it or leave it: could use inlined format args here:
```suggestion
                    (Some(task_id), Some(name)) => format!("{task_id} ({name})"),
```

Reply:
I really thought that in-line format args was a newer feature, but I looked it up (actually, I kind of just asked someone) and realised it's from 1.58. I'll apply this.